src/WebJobs.Extensions.ApiHub/Config/ApiHubConfiguration.cs

```diff
@@ -86,7 +88,7 @@ string IFileTriggerStrategy<ApiHubFile>.GetPath(ApiHubFile file)
         {
             if (file == null)
             {
-                throw new ArgumentNullException("file");
+                throw new ArgumentNullException(nameof(file));
```

Comment:
Yeah, it's on my todo list to ask to have the CI machines updated, but higher pri things keep coming up. We use the ASP.NET CI lab, so we don't own it, and other stuff runs on those machines, which complicates things.